pkg/llama/loader.go

```diff
@@ -82,14 +89,7 @@ func Load(path string) error {
 // Init is a convenience function to handle initialization of llama.cpp.
 func Init() {
 	BackendInit()
-
-	if os.Getenv("YZMA_LIB") != "" {
-		GGMLBackendLoadAllFromPath(os.Getenv("YZMA_LIB"))
-
-		return
-	}
-
-	GGMLBackendLoadAll()
+	GGMLBackendLoadAllFromPath(LibPath())
```

Comment:
I see what you did, the llama.Load function already accepted the libPath so now you have that as global state and use that in the call to GGMLBackendLoadAllFromPath.

I'm not sure you need the LibPath function since the user of the API already knows where it is, they are giving it to you. You could just use the libPath variable with the GGMLBackendLoadAllFromPath function.